meta-lmp-base/recipes-core/base-files/base-files-issue.bb

```diff
@@ -0,0 +1,45 @@
+FILESEXTRAPATHS:prepend := ":${COREBASE}/meta/recipes-core/base-files/base-files:"
+
+def get_pv(d):
+    import re
+    corebase = d.getVar('COREBASE')
+    bb_dir = os.path.join(corebase, 'meta', 'recipes-core', 'base-files')
+    if os.path.isdir(bb_dir):
+        re_bb_name = re.compile(r"base-files_([0-9.]*)\.bb")
+        for bb_file in os.listdir(bb_dir):
+            result = re_bb_name.match(bb_file)
+            if result:
+                return result.group(1)
+    bb.fatal("Cannot find base-files recipe in %s" % bb_dir)
+
+PV := "${@get_pv(d)}"
+
+require recipes-core/base-files/base-files_${PV}.bb
+
+SUMMARY += " (only contains /etc/issue*)"
+DESCRIPTION += " This package only install the /etc/issue* that have the LmP release (including the build number) to avoid initrd images rebuilding"
+
+SRC_URI = "file://issue.net \
+           file://issue \
+           file://licenses/GPL-2 \
+"
+
+pkg_preinst:${PN} () {
+    :
+}
+
+do_install () {
+    install -d ${D}${sysconfdir}
+    do_install_basefilesissue
+    return 0
```

Comment:
is return 0 needed here?

Reply:
I have no ideia how this got here, I don't see the reason for this.
But will do some tests because this PR It's been a while and I don't remember.

Reply:
The reason fo this is because we only need to call the `do_install_basefilesissue` as anyother files was already been installed.

This snip from the main recipe will be executed as well because of the `require recipes-core/base-files/base-files_${PV}.bb` and we don't carry the `${WORKDIR}/nsswitch.conf`  so it will fail.
```
do_install:append:libc-glibc () {
	install -m 0644 ${WORKDIR}/nsswitch.conf ${D}${sysconfdir}/nsswitch.conf
}
```

Reply:
Added a comment about this